mcu/atmega-hal/src/spi.rs

```diff
@@ -95,13 +115,3 @@ avr_hal_generic::impl_spi! {
     miso: port::PB6,
     cs: port::PB4,
 }
-
-#[cfg(any(feature = "atmega8"))]
-avr_hal_generic::impl_spi! {
-    hal: crate::Atmega,
-    peripheral: crate::pac::SPI,
-    sclk: port::PB5,
-    mosi: port::PB3,
-    miso: port::PB4,
-    cs: port::PB2,
-}
```

Comment:
Nice catch!  I took the liberty to merge this change already in commit b2be3f950c5a ("atmega-hal: Drop redundant atmega8 macro invocation").  Please rebase your PR on latest `main` when you send the next iteration :)